Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.h

```diff
@@ -70,12 +85,23 @@ class JSWebAssemblyArray final : public JSNonFinalObject {
 
     DECLARE_VISIT_CHILDREN;
 
-    Wasm::Type elementType() const { return m_elementType; }
+    Wasm::FieldType elementType() const { return m_elementType; }
     size_t size() const { return m_size; }
 
-    EncodedJSValue get(uint32_t index)
-    {
-        switch (m_elementType.kind) {
+    const FixedVector<uint8_t>& payload() const { return m_payload8; }
+
+    EncodedJSValue get(uint32_t index) {
```

Comment:
Minor nit: I think the webkit style checker may complain on the brace here, you'll want to run `Tools/Scripts/check-webkit-style` at some point to catch these (I won't generally point these out but wanted to mention the existence of that tool; it will also run if you do `git webkit pr` to make a PR so you don't necessarily need to run it explicitly).